### ISSUE TITLE
Run as non-root user

### ISSUE THREAD

**Issue by j-zimnowoda** [okteto/cert-manager-webhook-civo#22]
Hi there,
We are testing the integration of this webhook with [otomi-core](https://github.com/redkubes/otomi-core) project.

Would you consider running it as a non-root user ?
It would strengthen the security posture and allow to define a proper security context in the `values.yaml` file.
E.g.:
```
# Pod Security Context
# ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
securityContext:
  runAsNonRoot: true
  seccompProfile:
    type: RuntimeDefault

# Container Security Context to be set on the controller component container
# ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
containerSecurityContext:
  allowPrivilegeEscalation: false
  capabilities:
    drop:
    - ALL
  readOnlyRootFilesystem: true
  runAsNonRoot: true

```
Cheers!

**Comment by rberrelleza** [comment]
Yes, this would be great. It's been on my mind for a while. @j-zimnowoda if you have this working, would you mind sending a PR?

**Comment by j-zimnowoda** [comment]
sure, just opened PR

**Comment by rberrelleza** [comment]
fixed by #23
